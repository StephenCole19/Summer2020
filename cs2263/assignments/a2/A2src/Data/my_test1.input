u 1
o
o
x

