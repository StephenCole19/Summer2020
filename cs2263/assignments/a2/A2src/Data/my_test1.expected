1
1
failed pop
Successfully executed 2 instructions
0 elements
